module/docs/entra-powershell-beta/Microsoft.Graph.Entra.Beta/New-EntraBetaApplicationPasswordCredential.md

```diff
@@ -85,13 +99,20 @@ This command creates new password credential for specified application.
 ### Example 4: Create a password credential using EndDate parameter
 
 ```powershell
-New-EntraBetaApplicationPasswordCredential -ObjectId 'tttttttt-0000-2222-0000-aaaaaaaaaaaa'-EndDate (get-date).AddYears(2)
+Connect-Entra -Scopes 'Application.ReadWrite.All' #Delegated Permission
+Connect-Entra -Scopes 'Application.ReadWrite.OwnedBy' #Application Permission
+$params = @{
+    ObjectId = 'tttttttt-0000-2222-0000-aaaaaaaaaaaa'
+    EndDate = (Get-Date).AddYears(2)
+}
+
+New-EntraBetaApplicationPasswordCredential @params
```

Comment:
In example description we need t to add used parameters description for ObjectId and EndDate